DOCS-3930 Moving Around the Synthetic Testing Tunnel (#15316)
DataDog/documentation: content/en/continuous_testing/cicd_integrations/testing_tunnel.md

---

## Overview

The Synthetic testing tunnel creates short lived secure connections between your internal environments and the Datadog infrastructure, allowing you to swiftly trigger Synthetic tests on your private applications.  

> Datadog recommends using the testing tunnel if you are willing to launch Synthetics tests from your CI/CD pipeline or against local versions of your application without deploying a dedicated and long lasting probing system (such as [private locations][1]). The testing tunnel is helpful if you want to trigger tests on ephemeral cloud environments.

Reviewer: ```suggestion
Datadog recommends using the testing tunnel if you need to launch Synthetics tests against local versions of your application without deploying a dedicated and long lasting probing system (such as [private locations][1]). The testing tunnel can be used to trigger tests on ephemeral cloud environments.
```
